config/jobs/kubernetes-sigs/sig-windows/generate-presubmits.sh

```diff
@@ -0,0 +1,293 @@
+      ;;
+    1.19)
+      dockershim_api_model="https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_release_1_19.json"
+      containerd_api_model="https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_containerd_1_19.json"
+      ;;
+    1.20)
+      dockershim_api_model="https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_release_staging.json"
+      containerd_api_model="https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_release_1_20.json"
+      ;;
+    1.21)
+      ;;
+    *)
+      branch="master"
+      orchestrator_release="1.21"
+      ;;
+  esac
+
+  cat >"${output}" <<EOF
+# generated by ./config/jobs/kubernetes-sigs/sig-windows/generate-presubmits.sh.
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-aks-engine-windows-dockershim
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    path_alias: k8s.io/kubernetes
+    branches:
+    - ${branch}
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-azure-windows: "true"
+      preset-windows-repo-list-master: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-windows-private-registry-cred: "true"
+    spec:
+      containers:
+      - image: ${kubekins_test_image}-${release}
+        command:
+        - runner.sh
+        - kubetest
+        args:
+        # Generic e2e test args
+        - --test
+        - --up
+        - --down
+        - --build=quick
+        - --dump=\$(ARTIFACTS)
+        # Azure-specific test args
+        - --deployment=aksengine
+        - --provider=skeleton
+        - --aksengine-admin-username=azureuser
+        - --aksengine-admin-password=AdminPassw0rd
+        - --aksengine-creds=\$(AZURE_CREDENTIALS)
+        - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+        - --aksengine-public-key=\$(K8S_SSH_PUBLIC_KEY_PATH)
+        - --aksengine-private-key=\$(K8S_SSH_PRIVATE_KEY_PATH)
+        - --aksengine-winZipBuildScript=\$(WIN_BUILD)
+        - --aksengine-orchestratorRelease=${orchestrator_release}
+        - --aksengine-template-url=${dockershim_api_model}
+        - --aksengine-win-binaries
+        - --aksengine-deploy-custom-k8s
+        - --aksengine-agentpoolcount=2
+        # Specific test args
+        - --test_args=--node-os-distro=windows --docker-config-file=\$(DOCKER_CONFIG_FILE) --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-network\].EndpointSlice --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Slow\]|\[alpha\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application|device.plugin.for.Windows
```

Comment:
I know we had trouble trying to parametrize this in the past but since this is in a script it might work better?  Could we put this at the top and use across the versions below?